charts/chainflip-node/values.yaml

```diff
@@ -42,6 +42,11 @@ engine:
       rpc:
         ws_endpoint: ""
         http_endpoint: ""
+    arb:
+      private_key_file: /etc/chainflip/keys/ethereum_key_file
```

Comment:
maybe just a comment so it's clear this isn't a mistake but intentional, we want to use the same key here for simplicity

Reply:
true. Done.